scripts/test

```diff
@@ -85,15 +85,32 @@ function pretests () {
 # | cryptography               | 39.0.2    | >=0.8,<41.0.3            | 60224    |
 # | cryptography               | 39.0.2    | >=0.8,<41.0.3            | 60225    |
 # | cryptography               | 39.0.2    | >=0.8,<41.0.3            | 60223    |
-# | pip                        | 23.2.1    | <23.3                    | 62044    |
 # | py                         | 1.11.0    | <=1.11.0                 | 51457    |
 # | safety                     | 1.10.3    | <2.2.0                   | 51358    |
 # | sentry-sdk                 | 1.6.0     | <1.14.0                  | 53812    |
-# | urllib3                    | 1.26.16   | <1.26.17                 | 61601    |
-# | urllib3                    | 1.26.16   | <1.26.18                 | 61893    |
-# | validators                 | 0.20.0    | >=0.11.0,<0.21.0         | 61924    |
+
+# | flask                      | 1.1.4     | <2.2.5                   | 55261    |
+# | flask-security             | 3.0.0     | <3.1.0                   | 45183    |
+# | flask-security             | 3.0.0     | >0                       | 44501    |
+# | pillow                     | 9.5.0     | <10.0.1                  | 61489    |
+# | sqlalchemy                 | 1.3.24    | <2.0.0b1                 | 51668    |
+# | sqlalchemy-utils           | 0.35.0    | >=0.27.0                 | 42194    |
+# | wtforms                    | 2.3.3     | <3.0.0a1                 | 42852    |
+# | werkzeug                   | 1.0.1     | <2.2.3                   | 53325    |
+# | werkzeug                   | 1.0.1     | <2.2.3                   | 53326    |
+# | celery                     | 5.1.2     | <5.2.0                   | 42498    |
+# | celery                     | 5.1.2     | <5.2.2                   | 43738    |
+# | click                      | 7.1.2     | <8.0.0                   | 47833    |
+# | cryptography               | 39.0.2    | <41.0.0                  | 59062    |
```

Comment:
Could you clean up this list? Most elements are listed twice.

Reply:
Can not clean up the list because the error number is not the same.